packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -1066,16 +1072,7 @@ describe("normalizeAndValidateConfig()", () => {
 			expect(diagnostics.hasErrors()).toBe(false);
 			expect(diagnostics.renderWarnings()).toMatchInlineSnapshot(`
 				"Processing wrangler configuration:
-				  - \\"unsafe\\" fields are experimental and may change or break at any time.
-				  - In wrangler.toml, you have configured [durable_objects] exported by this Worker (CLASS1), but no [migrations] for them. This may not work as expected until you add a [migrations] section to your wrangler.toml. Add this configuration to your wrangler.toml:
-
-				      \`\`\`
-				      [[migrations]]
-				      tag = \\"v1\\" # Should be unique for each entry
-				      new_classes = [\\"CLASS1\\"]
-				      \`\`\`
-
-				    Refer to https://developers.cloudflare.com/durable-objects/reference/durable-objects-migrations/ for more details."
```

Comment:
Since this warning is an error with the rest of my changes, I'm adding a migration to the test so that this output. This message will still be populated in some cases. See packages/wrangler/src/config/validation.ts.